src/status_im/ui/screens/wallet/utils.cljs

```diff
@@ -0,0 +1,5 @@
+(ns status-im.ui.screens.wallet.utils
+  (:require [status-im.utils.money :as money]))
+
+(defn format-amount [amount decimals]
+  (money/to-fixed (money/token->unit (or amount (money/bignumber 0)) decimals)))
```

Comment:
Too many expressions nested, i find this easier to read:
```clj
(defn format-amount [amount decimals]
  (-> amount
      (or (money/bignumber 0))
      (money/token->unit decimals)
      money/to-fixed))
```

Reply:
Makes sense!